tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/SimpleOpenWireTest.java

```diff
@@ -1467,6 +1468,27 @@ public void testTempQueueSendAfterConnectionClose() throws Exception {
       }
    }
 
+   @Test
+   public void testNotificationProperties() throws Exception {
+      try (TopicConnection topicConnection = factory.createTopicConnection()) {
+         TopicSession topicSession = topicConnection.createTopicSession(false, Session.AUTO_ACKNOWLEDGE);
+         Topic notificationsTopic = topicSession.createTopic("activemq.notifications");
+         TopicSubscriber subscriber = topicSession.createSubscriber(notificationsTopic);
+         List<Message> receivedMessages = new ArrayList<>();
+         subscriber.setMessageListener(receivedMessages::add);
+         topicConnection.start();
+
+         while (receivedMessages.size() == 0) {
```

Comment:
More as learning exercise (since I could change this myself)...

you could use this:
```java
import org.apache.activemq.artemis.tests.util.Wait;
...

         Wait.waitFor(() -> receivedMessages.size() > 0);

         Assert.assertTrue(receivedMessages.size() > 0);
```